apps/authentication/api/tests/user_tests.py

```diff
@@ -0,0 +1,12 @@
+from django.core.urlresolvers import reverse
```

Comment:
`django.core.urlresolvers` is being deprecated in Django 2.
Would be nice to make the move to `django.urls` when making new stuff.

The package is mostly the same, and this would work in this case:
```
from django.urls import reverse
```

The rest will be taken care of in #2156